### PR TITLE
Update POPS constant to contain a dictionary of both exomes and genomes

### PR DESCRIPTION
**Pull request by klaricch**

### CODE REVIEW

**Review by mike-w-wilson — CHANGES_REQUESTED** [review]
Just one small change and then this LGTM!

File: gnomad/resources/grch38/gnomad.py [hunk 1]
```diff
@@ -80,17 +80,31 @@
 
 POPS = {
     "v3": ["afr", "ami", "amr", "asj", "eas", "fin", "nfe", "oth", "sas", "mid"],
```
[the remark this answers is not in the document]
Comment (klaricch) [reply gnomad/resources/grch38/gnomad.py:82]: ah good point

File: gnomad/utils/release.py [hunk 2]
```diff
@@ -35,7 +35,7 @@ def make_faf_index_dict(
     :param faf_meta: Global annotation containing the set of groupings for each element of the faf array
         (e.g., [{'group': 'adj'}, {'group': 'adj', 'pop': 'nfe'}])
     :param groups: List of sample groups [adj, raw]. Default is GROUPS
-    :param pops: List of sample global population names for gnomAD genomes. Default is POPS[CURRENT_MAJOR_RELEASE]
+    :param pops: List of sample global population names for gnomAD genomes. Default is POPS[CURRENT_MAJOR_RELEASE]["exomes"].
```
Comment (mike-w-wilson) [inline gnomad/utils/release.py:38]:
```suggestion
    :param pops: List of sample global population names for gnomAD data type. Default is POPS[CURRENT_MAJOR_RELEASE]["exomes"].
```

File: gnomad/utils/release.py [hunk 3]
```diff
@@ -73,7 +73,7 @@ def make_freq_index_dict(
     :param freq_meta: List containing the set of groupings for each element of the freq array
         (e.g., [{'group': 'adj'}, {'group': 'adj', 'pop': 'nfe'}])
     :param groups: List of sample groups [adj, raw]. Default is GROUPS
-    :param pops: List of sample global population names for gnomAD genomes. Default is POPS[CURRENT_MAJOR_RELEASE]
+    :param pops: List of sample global population names for gnomAD genomes. Default is POPS[CURRENT_MAJOR_RELEASE]["exomes"].
```
Comment (mike-w-wilson) [inline gnomad/utils/release.py:76]:
```suggestion
    :param pops: List of sample global population names for gnomAD data type. Default is POPS[CURRENT_MAJOR_RELEASE]["exomes"].
```

**Review by mike-w-wilson — APPROVED** [review]
Two small things then this looks good! Thanks for doing it!

**Pull request merged**